packages/office-addin-debugging/src/commands.ts

```diff
@@ -39,20 +53,41 @@ export async function start(manifestPath: string, appType: string | undefined, c
         const packager: string | undefined = command.packager || process.env.npm_package_scripts_packager;
         const packagerHost: string | undefined = command.PackagerHost || process.env.npm_package_config_packager_host;
         const packagerPort: string | undefined = command.PackagerPort || process.env.npm_package_config_packager_port;
+        const sourceBundleUrlComponents = new devSettings.SourceBundleUrlComponents(
+            command.sourceBundleUrlHost, command.sourceBundleUrlPort,
+            command.sourceBundleUrlPath, command.sourceBundleUrlExtension);
+
+        if (appPlatformToDebug === undefined) {
+            throw new Error("Please specify the application platform to debug.");
+        }
 
         if (appTypeToDebug === undefined) {
             throw new Error("Please specify the application type to debug.");
         }
 
+        if (manifestPath === "") {
+            manifestPath = determineManifestPath(appPlatformToDebug, devBuild);
+        }
+
         await startDebugging(manifestPath, appTypeToDebug, app, debuggingMethod, sourceBundleUrlComponents,
             devServer, devServerPort, packager, packagerHost, packagerPort, enableDebugging, enableLiveReload);
     } catch (err) {
         logErrorMessage(`Unable to start debugging.\n${err}`);
     }
 }
 
-export async function stop(manifestPath: string, command: commander.Command) {
+export async function stop(manifestPath: string, platform: string | undefined, command: commander.Command) {
     try {
+        const appPlatformToDebug: AppPlatform | undefined = parseAppPlatform(platform || process.env.npm_package_config_app_plaform_to_debug || AppPlatform.Win32);
```

Comment:
same comments about not needed App/application with platform; parsePlatform(), "Please specify the platform to debug.".